index.html

```diff
@@ -2355,13 +2355,23 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the sender of a pre-signed
+message to reveal parts of the message without revealing the entire message. For
+example, selectively disclosing a digital driver's license for the purposes of
+renting a car might involve the entity presenting the license to only reveal the
+issuing authority, license number, birthday, and authorized motor vehicle class.
+Note that in this case, the license number is correlatable information that is
+being shared but some amount of privacy is still being preserved because the
+driver's full name and address are not being shared.
```

Comment:
```suggestion
Selective disclosure is a technique where the sender of a pre-signed
message reveals only parts of the whole message without the receiver
losing confidence in their authenticity. For example, one might selectively
disclose a digital driver's license for the purpose of renting a car. This could
involve revealing only the issuing authority, license number, birthday, and
authorized motor vehicle class from the license. Note that in this case, the
license number is correlatable information, but some amount of privacy
is preserved because the driver's full name and address are not shared.
```

Reply:
Hmm, "without the receiver losing confidence in their authenticity" is a little unclear to me. Perhaps something like "....reveals only parts of the whole message without the receiver losing confidence in the authenticity of the parts"

or "....reveals only parts of the whole message while maintaining confidence in the authenticity of each of the parts"

Reply:
fwiw, I prefer Ted's suggestion over this one

Reply:
I've applied @TallTed's suggestion. @dlongley could you please re-review to make sure your suggestions made it in (based on Ted's changes).